packages/core/src/App/Containers/Layout/header/header.jsx

```diff
@@ -7,15 +7,15 @@ import DTraderHeader from './dtrader-header.jsx';
 import { connect } from 'Stores/connect';
 import { useLocation } from 'react-router-dom';
 
-const Header = ({ is_logged_in, is_pre_appstore }) => {
+const Header = ({ is_logged_in }) => {
     const { is_appstore } = React.useContext(PlatformContext);
     const { pathname } = useLocation();
     const trading_hub_routes =
         pathname === routes.traders_hub || pathname.startsWith(routes.cashier) || pathname.startsWith(routes.account);
 
     if (is_appstore) {
         return <DashboardHeader />;
-    } else if (is_logged_in && is_pre_appstore) {
+    } else if (is_logged_in) {
```

Comment:
DtraderHeader needs some adjustments regarding `old-ui` and `new-ui`

Reply:
Yeap. Working on it